### ISSUE TITLE
Lint not detected for shorthand properties zero values

### ISSUE THREAD

**Issue by sds** [brigade/scss-lint#37]
For example, `scss-lint` does not report a lint for the `0em` at the end of the `padding` property shorthand.

``` css
p {
  padding: 1.3em 1em 0em;
}
```

This is due to the regex used to detect these cases. It needs to be modified to support detecting more than one occurrence of a value on a line. 

It also does not work for arguments of mixins or functions, e.g.

``` css
@include my-mixin(0em);
```
